packages/main/src/MenuItem.ts

```diff
@@ -201,6 +230,26 @@ class MenuItem extends ListItem implements IMenuItem {
 	@i18n("@ui5/webcomponents")
 	static i18nBundle: I18nBundle;
 
+	/**
+	 * Defines whether `ui5-menu-item` is in selected state.
+	 *
+	 * **Note:** selected state is only taken into account when `ui5-menu-item` is added to `ui5-menu-item-group`
+	 * with `itemSelectionMode` other than `None`.
+	 * **Note:** A selected `ui5-menu-item` have selection mark displayed ad its end.
+	 * @default false
+	 * @public
+	 * @since 2.5.0
+	 */
+	@property({ type: Boolean })
+	set isSelected(value: boolean) {
```

Comment:
`isSelected` is not ok for a property name. It must be `selected`. Let's discuss refactoring possibilities to make `selected` not clash with the list item one.